### PR TITLE
fix(core/properties): Fix typings for nativeValueChange

### PR DESCRIPTION
**Pull request by speigg**

### CODE REVIEW

File: tns-core-modules/ui/core/properties/properties.d.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ export class Property<T extends ViewBase, U> {
     public readonly setNative: symbol;
     public readonly defaultValue: U;
     public register(cls: { prototype: T }): void;
-    public nativeValueChange(T, U): void;
+    public nativeValueChange(target:T, value:U): void;
```
Comment (manoldonev) [inline tns-core-modules/ui/core/properties/properties.d.ts]: First argument is generally referred as `owner` across the codebase: https://github.com/NativeScript/NativeScript/blob/7edf56197a179b13e1afe4c706aea4a99ac62aca/tns-core-modules/ui/core/properties/properties.ts#L70
Can you update this?

**Comment by vakrilov** [comment]
Hey @speigg 
I think this PR is almost ready to be merged. Do you mind implementing @manoldonev 's suggestion about the `owner` param and add this to `master`.

**Comment by speigg** [comment]
I made the changes directly on Github.com, as I don’t have the {N} repository downloded and setup for development. Anyway, the CI seems to be happy about the whitespace now.

**Comment by vakrilov** [comment]
test

**Pull request merged**